Dockerfiles/go-app-common.Dockerfile

```diff
@@ -12,5 +12,6 @@ FROM --platform=$TARGETPLATFORM ${BASE_IMAGE_SHA}
 ARG BINARY_TO_ADD
 
 ADD ${BINARY_TO_ADD} /usr/local/bin/
+RUN ln -s /usr/local/bin/${BINARY_TO_ADD} /entrypoint_executable
 
-ENTRYPOINT ["/usr/local/bin/${BINARY_TO_ADD}"]
+ENTRYPOINT ["/entrypoint_executable"]
```

Comment:
Optional: Why not `/usr/local/bin/entrypoint`, maybe even a script that runs the actual binary?

Reply:
Looks good to me, i will update to `/usr/local/bin/entrypoint`